### PR TITLE
[ZK-127] Add aggregator diagram

### PR DESCRIPTION
**Pull request by maxatdeltafi**
## Changes

- Add aggregator diagram

## TODOs
- Write some unit test

### CODE REVIEW

File: aggregator/README.md [hunk 1]
```diff
@@ -1,59 +1,70 @@
-Proof Aggregation
------
+## Proof Aggregation
 
 ![Architecture](./figures/architecture.png)
 
+![Function Diagram](./figures/diagram.jpg)
```
Comment (pjxthy) [inline aggregator/README.md:5]: diagram.jpg to function_calls.jpg ?

File: aggregator/aggregator.dio [hunk 2]
```diff
@@ -0,0 +1,161 @@
+<mxfile host="65bd71144e">
```
Comment (pjxthy) [inline aggregator/aggregator.dio:1]: aggregator/aggregator.dio -> to a jpg or png file? and add to readme

**Review by pjxthy — CHANGES_REQUESTED** [review]

File: aggregator/aggregator.dio (hunk 2 above)
Comment (maxatdeltafi) [reply aggregator/aggregator.dio:1]: its a vscode extension, the function_calls picture is screenshot from.

File: aggregator/README.md (hunk 1 above)
Comment (maxatdeltafi) [reply aggregator/README.md]: done

**Review by pjxthy — APPROVED** [review]

**Pull request merged**